### ISSUE TITLE
version numbers missing on macOS

### ISSUE THREAD

**Issue by core-code** [openMSX/openMSX#1121]
i've downloaded Version '0.14.0' but when selecting it in the Finder and using 'Get Info' it doesn't show any Version-number at all. i guess the **CFBundleVersion** and **CFBundleShortVersionString** entries in your **Info.plist** file are missing.
all apps on macOS are supposed to contain valid version information.

**Comment by MBilderbeek** [comment]
Patch welcome! I guess you need to modify build/package-darwin/app.mk

**Comment by core-code** [comment]
the Info.plist in openmsx-0.14.0-mac-x86_64-bin.dmg
contains this 
```
	<key>CFBundleShortVersionString</key>
	<string></string>
	<key>CFBundleVersion</key>
	<string></string>

```
so i guess the replacement of
%VERSION%
doesn't work properly

i am not really familiar with the syntax used in the app.mk file or with your particular setup but maybe replacing
PACKAGE_DETAILED_VERSION
with
PACKAGE_FULL
will improve things?

**Comment by mthuurne** [comment]
`PACKAGE_DETAILED_VERSION` was removed in a3bcd78b, so that was indeed the cause. That happened in 2009 though and until now no-one has noticed or reported it...

I committed something that should fix it. I no longer have a working Mac, but the script fragment does insert the version correctly into the Info.plist template when run on Linux, so it should work.

**Comment by core-code** [comment]
awesome, thanks

**Comment by mthuurne** [comment]
Fixed in the openmsx-0.14.0-365 [snapshot build](http://openmsx.fixato.net/builds/mac/x86_64/).

**Comment by core-code** [comment]
i can confirm that. thanks
